pkgs/servers/soft-serve/default.nix

```diff
@@ -21,9 +21,14 @@ buildGoModule rec {
 
   postInstall = ''
     wrapProgram $out/bin/soft \
```

Comment:
can you add a comment telling us why you are doing wrapProgram instead of patching it?
IIRC you mentioned something about scripts getting generated at runtime from a go template file.